tabs/onboard_logging.js

```diff
@@ -10,7 +10,14 @@ TABS.onboard_logging.initialize = function (callback) {
     var 
         self = this,
         saveCancelled, eraseCancelled;
+    newFC.purge(); // This should be noved to a general "change-tab" callback
 
+    BLACKBOX.listen(load_html);
+    DATAFLASH.listen(load_html);
+    PID_ADVANCED_CONFIG.listen(load_html);
```

Comment:
I think it needs a clear function to remove all listeners when changing tab. Or only allow one listener, might be a better option ? This is on the todo list.